### PR TITLE
DOC: Add examples to io.wavfile docstrings

### PR DESCRIPTION
**Pull request by Psirus**
#### Reference issue
Another small step wrt. #7168.

### CODE REVIEW

**Comment by larsoner** [comment]
Failure is real:

https://travis-ci.org/scipy/scipy/jobs/598066431

**Comment by Psirus** [comment]
I've used the `io.loadmat` docstring as a blueprint, should now be fixed.

File: scipy/io/wavfile.py [hunk 1]
```diff
@@ -225,6 +225,24 @@ def read(filename, mmap=False):
...
+    >>> data_dir = pjoin(dirname(sio.__file__), 'tests', 'data')
+    >>> wav_fname = pjoin(data_dir, 'test-44100Hz-2ch-32bit-float-be.wav')
+
+    Load the .wav file contents.
+
+    >>> samplerate, data = sio.wavfile.read(wav_fname)
+    >>> print(f"number of channels = {data.shape[1]}")
+    number of channels = 2
+    >>> print(f"length = {data.shape[0] / samplerate}s")
+    length = 0.01s
```
Comment (larsoner) [inline scipy/io/wavfile.py]: Is it worth doing a quick plot of the data, too?

**Comment by rgommers** [comment]
LGTM, merged. Thanks @Psirus!

**Pull request merged**